includes/gateways/class-wpsms-gateway-africastalking.php

```
@@ -8,6 +8,7 @@ class africastalking extends \WP_SMS\Gateway
 {
     private $wsdl_link = 'https://api.africastalking.com/version1';
     public $tariff = "https://africastalking.com";
+    public $documentUrl = 'https://wsms.io/docs/africastalking-gateway-setup';
```

PHPStan (2.2.7)

[error] Property WP_SMS\Gateway\africastalking::$documentUrl (bool) does not accept default value of type string.


(property.defaultValue)